src/bin/bat/clap_app.rs

```diff
@@ -59,6 +59,18 @@ pub fn build_app(interactive_output: bool) -> Command {
                      Use '--tabs' to control the width of the tab-placeholders.",
                 ),
         )
+        .arg(
+            Arg::new("caret-notation")
+                .long("caret-notation")
+                .alias("caret")
```

Comment:
I would prefer if we didn't make `--caret` a short form of `--caret-notation`. "caret notation" indeed seems to be the [official term](https://en.wikipedia.org/wiki/Caret_notation) for this, and I don't think `--caret` is clear enough on its own. I wouldn't rule out us adding an alias in the future, but since it is much easier to add stuff than to remote stuff, I would prefer if we started out without it.